sdk/spring-sdk/src/test/java/kalix/springsdk/testmodels/action/ActionsTestModels.java

```diff
@@ -54,6 +57,17 @@ public Action.Effect<Message> message(@RequestBody Message msg) {
     }
   }
 
+  public static class PostWithoutParamWithJWT extends Action {
+    @PostMapping("/message")
+    @JWT.JwtMethodOptions(
+        validate = {JwtMethodMode.BEARER_TOKEN},
```

Comment:
When the annotation field is an array and we have only one element, then the `{}` is optional.

Reply:
removed